ONSdigital/eq-questionnaire-runner
# Bug: Cross-section when rule dependencies not working as expected fix

### What is the context of this PR?
[Bug: Cross-section when rule dependencies not working as expected](Bug: Cross-section when rule dependencies not working as expected)

### How to review

With the current main branch 

- Launch phm-0001 schema 
- Answer questions until you get to the age question
- Answer No to are you aged 16 or above.
- You will eventually get to a question that says Have you ever smoked cigarettes regularly?
This question should not have been in the path as it was dependent on the age.

With fix branch 

- Launch phm-0001 schema 
- Answer questions until you get to the age question
- Answer No to are you aged 16 or above.
- You will not get to a question that says Have you ever smoked cigarettes regularly?

### Checklist
